scripts/controls.js

```diff
@@ -27,8 +27,8 @@ export default class Controls {
 
   get _convenientEffectsAppButton() {
     return {
-      name: 'convenient-effects',
-      title: 'Add Convenient Effects',
+      name: game.i18n.localize("Controls.convenienteffects"),
```

Comment:
I don't think this one needs to be localized

Reply:
Noted! Will undo